src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1656,68 +1207,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         const auto point = args.GetCurrentPoint(*this);
         const auto props = point.Properties();
-        const TerminalInput::MouseButtonState state{ props.IsLeftButtonPressed(), props.IsMiddleButtonPressed(), props.IsRightButtonPressed() };
-        auto result = _DoMouseWheel(point.Position(),
-                                    ControlKeyStates{ args.KeyModifiers() },
-                                    point.Properties().MouseWheelDelta(),
-                                    state);
+        const TerminalInput::MouseButtonState state{ props.IsLeftButtonPressed(),
+                                                     props.IsMiddleButtonPressed(),
+                                                     props.IsRightButtonPressed() };
+        auto mousePosition = point.Position();
+
+        auto result = _interactivity->MouseWheel(ControlKeyStates{ args.KeyModifiers() },
+                                                 point.Properties().MouseWheelDelta(),
+                                                 _GetTerminalPosition(mousePosition),

```

Comment:
Isn't this interactivity's job?

Reply:
I have this as _no_, because the interactivity has no concept of padding. So only the control can figure that out. There's only the one that takes the pixel position, and I bet if we tried harder we could get rid of that one too.

Reply:
Ah yeah but padding is a pixel delta applied to the pixels before we send the position to the Interactivity layer

Reply:
I legit feel like the only conduit between UWP and Interact should be pixel data here— that’s how it handles selections and knows how far the mouse moved and stuff, right? Nobody else at the UWP layer needs to get the font grid cell size or calculate terminal positions right now..

Reply:
Okay, that's not the worst idea. But it's the last thing I'm gonna come back around to on this PR